server/src/main/java/org/apache/cloudstack/vm/UnmanagedVMsManagerImpl.java

```diff
@@ -818,9 +806,9 @@ private UserVm migrateImportedVM(HostVO sourceHost, VirtualMachineTemplate templ
                 continue;
             }
             boolean poolSupportsOfferings = storagePoolSupportsDiskOffering(diskProfileStoragePool.second(), dOffering);
-            if (poolSupportsOfferings && profile.getType() == Volume.Type.ROOT) {
-                poolSupportsOfferings = storagePoolSupportsServiceOffering(diskProfileStoragePool.second(), serviceOffering);
-            }
+            //FR123 get confirmation from Abhishek if (poolSupportsOfferings && profile.getType() == Volume.Type.ROOT) {
+            //    poolSupportsOfferings = storagePoolSupportsServiceOffering(diskProfileStoragePool.second(), serviceOffering);
+            //}
```

Comment:
As git will keep the history, we can remove the code and move the comment from code to commit message. If it is really needed in code, we can turn it into a javadoc.